package/MDAnalysis/analysis/hbonds/hbond_analysis.py

```diff
@@ -1037,17 +1038,12 @@ def _get_timestep():
     @staticmethod
     def calc_angle(d, h, a, box=None):
         """Calculate the angle (in degrees) between two atoms with H at apex."""
-        v1= d.position
-        v2= h.position
-        v3= a.position
-        angle= calc_angles(v1[None, :], v2[None, :], v3[None, :], box=box)
-    
-        return np.rad2deg(angle[0])
+        return distances.calc_angle(d.position, h.position, a.position, box)
 
     @staticmethod
     def calc_eucl_distance(a1, a2, box=None):
```

Comment:
Can you get rid of the method completely and just call `distances.calc_distance()` in the code?